plutus-e2e-tests/test/Helpers.hs

```diff
@@ -144,6 +165,52 @@ startTestnet era testnetOptions base tempAbsBasePath' = do
   liftIO $ IO.setEnv "CARDANO_NODE_SOCKET_PATH" socketPathAbs -- set node socket environment for Cardano.Api.Convenience.Query
   pure (localNodeConnectInfo, pparams, networkId)
 
+connectToLocalNode :: C.CardanoEra era
+  -> LocalNodeOptions
+  -> FilePath
+  -> H.Integration (C.LocalNodeConnectInfo C.CardanoMode, C.ProtocolParameters, C.NetworkId)
+connectToLocalNode era localNodeOptions tempAbsPath = do
+  localEnvDir <- return $ localEnvDir localNodeOptions
```

Comment:
Maybe use a `let localEnvDir = ..` here?

Reply:
must be different name so will go with `localEnvDir'`